test/TestUtilities/Test.Utility/DebuggerUtils.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Test.Utility
+{
+    public class DebuggerUtils
```

Comment:
It looks like it is for core only. How about adding ifdefs for the framework and doing a debugger.launch for desktop?

the class should still be static also.